docs/_layouts/landing.html

```diff
@@ -19,17 +19,17 @@ <h3>Work on Features, not Security</h3>
 						<i class="fa fa-angle-double-right btn-arrow-right" aria-hidden="true"></i>
 						<h3>Prevent Loss and Theft of Secrets</h3>
 					</div>
-					<p class="inner-text-description">Credentials carry privilege. Don't allow your applications to manage any more sensitive information than necessary.<br>With the Secretless Broker, your applications can connect securely to target services, without becoming part of the secrets management threat surface</p>
+					<p class="inner-text-description">Credentials carry privilege so don't allow your applications to manage any more sensitive information than necessary.<br>With the Secretless Broker, your applications can connect securely to target services, without becoming part of the secrets management threat surface.</p>
 				</div>
 			</div>
 			<div class="col-md-4 about-subitem">
 				<div class="inner">
 					<div class ="in-line-frame">
 						<i class="fa fa-angle-double-right btn-arrow-right" aria-hidden="true"></i>
-						<h3>Handle Secrets Rotation Seamlessly</h3>
+						<h3>Handle Secret Rotations Seamlessly</h3>
 					</div>
 					<p class="inner-text-description">
-					It's hard for applications to respond to changing secrets. Until now, applications required additional logic to handle secrets rotation. With the Secretless Broker, no code changes or application logic is necessary - the Secretless Broker handles it all for you.</p>
+					Until now, applications usually required additional logic and disruptive restarts to handle credential rotation. With the Secretless Broker, no code changes are necessary and no application restarts are needed for secrets rotations - the Secretless Broker handles it all for you!</p>
```

Comment:
`secret rotations` instead of `secrets rotations`?

Reply:
Yup - that was explicitly mentioned in the issue